### PR TITLE
Removing unnecessary CNVS variable overrides.

### PR DESCRIPTION
**Pull request by ashenden**
Notable changes:
* Remove CNVS var overrides in `styles/vendors` directory that were doing nothing.
* Moved `icon-tiny` to main application styles and out of `styles/vendors/cnvs` directory.

### CODE REVIEW

File: src/styles/components/icons/shapes/icon-tiny/styles.less [hunk 1]
```diff
@@ -0,0 +1,78 @@
+& when (@icon-tiny-enabled) {
```
Comment (nLight) [inline src/styles/components/icons/shapes/icon-tiny/styles.less]: We're missing a top level `& when (@icon-enabled)` condition here. Is it by intention?

File: src/styles/components/typography/styles.less [hunk 2]
```diff
@@ -114,10 +114,6 @@
     color: @link-color;
   }
 
-  .text-small {
-    font-size: 0.7rem;
```
Comment (nLight) [inline src/styles/components/typography/styles.less]: .small in CNVS is actually a bit bigger `@small-font-size: 0.8rem;` are you fine with having it a bit bigger in this place?

**Comment by nLight** [comment]
I'd love to merge this one! Please answer my questions.

File: src/styles/components/icons/shapes/icon-tiny/styles.less (hunk 1 above)
Comment (ashenden) [inline src/styles/components/icons/shapes/icon-tiny/styles.less]: @nLight yes, that is intentional.  The check is only on the component variant itself, following the pattern set for similar icons in CNVS.

File: src/styles/components/typography/styles.less (hunk 2 above)
Comment (ashenden) [inline src/styles/components/typography/styles.less]: Good question.  Let's stick with the 0.8rem def.

**Comment by ashenden** [comment]
@nLight followed up on your comments.

**Pull request merged**